src/components/v5/common/ActionSidebar/partials/Motions/Motions.tsx

```diff
@@ -96,164 +108,196 @@ const Motions: FC<MotionsProps> = ({ transactionId }) => {
+            motionStateEnum ? (
+              <MotionCountDownTimer
+                motionState={motionStateEnum}
+                motionId={motionId}
+                motionStakes={motionStakes}
+                refetchMotionState={refetchMotionState}
+              />
+            ) : undefined,
+        },
+      },
+      {
+        key: NetworkMotionState.Submit,
+        content: (
+          <VotingStep
+            actionData={action as MotionAction}
+            startPollingAction={startPollingForAction}
+            stopPollingAction={stopPollingForAction}
+            transactionId={transactionId}
+          />
+        ),
+        heading: {
+          label: formatText({ id: 'motion.voting.label' }) || '',
+          decor:
+            motionStateEnum === MotionState.Voting && motionStakes ? (
+              <MotionCountDownTimer
+                motionState={motionStateEnum}
+                motionId={motionId}
+                motionStakes={motionStakes}
+                refetchMotionState={refetchMotionState}
+              />
+            ) : undefined,
+        },
+        isOptional: !isFullyStaked,
+        isHidden: motionStakedAndFinalizable,
+      },
+      {
+        key: NetworkMotionState.Reveal,
+        content: (
+          <RevealStep
+            motionData={motionData}
+            startPollingAction={startPollingForAction}
+            stopPollingAction={stopPollingForAction}
+            transactionId={transactionId}
+          />
+        ),
+        heading: {
+          label: formatText({ id: 'motion.reveal.label' }) || '',
+          decor:
+            motionStateEnum === MotionState.Reveal && motionStakes ? (
+              <MotionCountDownTimer
+                motionState={motionStateEnum}
+                motionId={motionId}
+                motionStakes={motionStakes}
+                refetchMotionState={refetchMotionState}
+              />
+            ) : undefined,
+        },
+        isHidden: motionStakedAndFinalizable,
+      },
+      {
+        key: CustomStep.StakedMotionOutcome,
+        content: <div />,
+        heading: {
+          iconName:
+            (motionStateHistory?.hasPassed && 'thumbs-up') ||
+            (motionStateHistory?.hasFailed && 'thumbs-down') ||
+            (motionStateHistory?.hasFailedNotFinalizable && 'thumbs-down') ||
+            '',
+          label:
+            (motionStateHistory?.hasPassed &&
+              formatText({ id: 'motion.passed.label' })) ||
+            (motionStateHistory?.hasFailed &&
+              formatText({ id: 'motion.failed.label' })) ||
+            (motionStateHistory?.hasFailedNotFinalizable &&
+              formatText({ id: 'motion.failed.label' })) ||
+            formatText({ id: 'motion.outcome.label' }) ||
+            '',
+          className: clsx({
+            '!bg-base-white !text-purple-400 border-purple-400':
+              motionStateHistory?.hasPassed,
+            '!bg-base-white !text-red-400 border-red-400':
```

Comment:
As far as I can tell, it does indeed pick it up as the classes get applied properly. 

![Screenshot from 2023-11-29 19-04-17](https://github.com/JoinColony/colonyCDapp/assets/1193222/ed9159be-cd73-48cf-bae1-0ad4a2c2fa02)
![Screenshot from 2023-11-29 19-04-43](https://github.com/JoinColony/colonyCDapp/assets/1193222/b9de8345-7e99-4adf-b1b8-b5987c238b04)

Reply:
That the CSS is applied is not proof that it actually picks it up.
I'm pretty sure this just works because we use these classes in other places.
Try using a class that's not used anywhere else (unlikely, I know) - I think this won't be applied.
I suggest to at least add a todo here to resolve this in the future.